src/main/java/com/enderio/base/common/config/common/BlocksConfig.java

```diff
@@ -4,28 +4,19 @@
 
 public class BlocksConfig {
     public final ForgeConfigSpec.ConfigValue<Float> BROKEN_SPAWNER_DROP_CHANCE;
-
     public final ForgeConfigSpec.ConfigValue<Float> EXPLOSION_RESISTANCE;
-
     public final ForgeConfigSpec.ConfigValue<Float> DARK_STEEL_LADDER_BOOST;
 
-    public final ForgeConfigSpec.ConfigValue<String> VACUUM_CHEST_RANGE_COLOR;
-    public final ForgeConfigSpec.ConfigValue<String> XP_VACUUM_RANGE_COLOR;
-
     public BlocksConfig(ForgeConfigSpec.Builder builder) {
         builder.push("blocks");
 
-        builder.push("brokenSpawner");
+        builder.push("brokenSpawner"); //TODO is this push needed?
```

Comment:
The clarity here is nice because otherwise it'd be a random `dropChance` key with no context.